apps/hobbies-helsinki/src/domain/search/eventSearch/filterSummary/FilterSummary.tsx

```diff
@@ -82,6 +84,7 @@ const FilterSummary: React.FC<Props> = ({ onClear }) => {
       dateTypes: getFilteredList('dateType', dateTypes),
       divisions: getFilteredList('division', divisions),
       end: type === 'date' ? null : end,
+      helsinkiOnly: type === 'helsinkiOnly' ? undefined : helsinkiOnly,
```

Comment:
Why wouldn't you use the HELSINKI_ONLY constant here?

Reply:
Taken into account in ea2b90da2f5c23a2b3a06d4586b31e6e964c9caa. The `type` variable comparison was left unchanged as the type of the variable `type` is `FilterType` instead of a constant from `EVENT_SEARCH_FILTERS`.